app/src/main/java/io/pslab/models/PSLabSensor.java

```diff
@@ -335,65 +339,21 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.play_data:
                 playingData = !playingData;
                 if (!startedPlay) {
-                    if (getSensorFragment() instanceof LuxMeterDataFragment) {
-                        ((LuxMeterDataFragment) getSupportFragmentManager()
-                                .findFragmentByTag(getSensorName())).playData();
-                    } else if (getSensorFragment() instanceof BaroMeterDataFragment) {
-                        ((BaroMeterDataFragment) getSupportFragmentManager()
-                                .findFragmentByTag(getSensorName())).playData();
-                    } else if (getSensorFragment() instanceof GyroscopeDataFragment) {
-                        ((GyroscopeDataFragment) getSupportFragmentManager()
-                                .findFragmentByTag(getSensorName())).playData();
-                    } else if (getSensorFragment() instanceof CompassDataFragment) {
-                        ((CompassDataFragment) getSupportFragmentManager()
-                                .findFragmentByTag(getSensorName())).playData();
-                    } else if (getSensorFragment() instanceof AccelerometerDataFragment) {
-                        ((AccelerometerDataFragment) getSupportFragmentManager()
-                                .findFragmentByTag(getSensorName())).playData();
-                    } else if (getSensorFragment() instanceof ThermometerDataFragment){
-                        ((ThermometerDataFragment) getSupportFragmentManager()
-                        .findFragmentByTag(getSensorName())).playData();
-                    } else if (getSensorFragment() instanceof GasSensorDataFragment){
-                        ((GasSensorDataFragment) getSupportFragmentManager()
-                                .findFragmentByTag(getSensorName())).playData();
-                    } else if (getSensorFragment() instanceof DustSensorDataFragment) {
-                        ((DustSensorDataFragment) getSupportFragmentManager()
-                                .findFragmentByTag(getSensorName())).playData();
-                    } else if(getSensorFragment() instanceof SoundMeterDataFragment) {
-                        ((SoundMeterDataFragment) getSupportFragmentManager()
-                                .findFragmentByTag(getSensorName())).playData();
+                     fragment=getSupportFragmentManager()
+                            .findFragmentByTag(getSensorName());
+
+                    if (fragment instanceof OperationCallback){
+                        ((OperationCallback) fragment).playData();
```

Comment:
Reformat the code here. You can use automatic option in Android studio. It will add missing spaces and indents.